app/indexers/hyrax/permission_indexer.rb

```diff
@@ -13,7 +13,7 @@ module PermissionIndexer
     def to_solr
       super.tap do |index_document|
         config      = Hydra.config.permissions
-        permissions = Hyrax::PermissionManager.new(resource: resource)
+        permissions = resource.permission_manager || Hyrax::PermissionManager.new(resource: resource)
```

Comment:
i think if this is right, just `resource.permission_manager` should do the trick (it builds one exactly like the `||` path if it's not present, and the second branch should be unreachable as is).

Reply:
👍